src/pages/Facility/overview.tsx

```diff
@@ -35,14 +37,14 @@ export function FacilityOverview({ facilityId }: FacilityOverviewProps) {
   ];
 
   return (
-    <div className="">
-      <div className="container p-6 mx-auto space-y-8">
+    <Page title="">
+      <div className="container mx-auto space-y-8">
         {/* Welcome Header */}
-        <div className="p-6 bg-white rounded-lg shadow-sm">
+        <div className="rounded-lg">
           <div className="flex items-center gap-4 mb-4">
             <div>
-              <h1 className="text-2xl font-bold text-gray-900">
-                Hey {user?.first_name}
+              <h1 className="text-4xl font-bold text-gray-900">
+                {t("hey_user", { user: user?.first_name })}
```

Comment:
We will need to account for sufix and prefix for a user. The doctors will need to see `Dr.`